docs/essentials/collections/selectingaCollectionClass.md

```diff
@@ -1 +1,69 @@
+# Selecting a Collection Class
```

Comment:
After reading the whole section carefully, it looks like the entire section focuses on the .NET 1.x collections. I'd recommend replacing the 1.x collections with the preferred generic version